src/site/xdoc/server/config-users.xml

```diff
@@ -41,8 +41,11 @@
         <li>username should not be longer than 255 characters</li>
         <li>username can not contain '/'</li>
         <li>username can not contain multiple domain delimiter('@')</li>
-        <li>A username can have only local part. E.g.'myUser'</li>
```

Comment:
`A username can have only local part when virtualHosting is enabled. E.g.'myUser'`